src/api_client/api_client.js

```diff
@@ -38,7 +38,7 @@ const defaultOptions = {
 
 const commonHeaders = {
 	'Content-Type': 'application/json',
-	os: 'lisk-js-api',
+	'User-Agent': 'lisk-js/1.0',
```

Comment:
@willclarktech This change is good but should't we change it to **LiskElements**? Or that will be done in one transition? 

Also one change I could suggest here. The standard `User-Agent` sting as following pattern: 

```
[ProductName]/[version] ([system and browser information]) [platform] ([platform details]) [extensions].
```

So here I believe; 

```
ProductName: LiskElements
Version: 1.0
System Information: If we can add we should 
Platform: We can add this information easily 
```

One last thing around all `UserAgents` I see every one uses `CamelCase` for Product Name, should not we follow it as well? e.g. Mozilla, AppleWebKit, Googlebot

If you think its not a browser, even then custom agents follow the same pattern.e.g. 

```
Googlebot/2.1 (+http://www.google.com/bot.html)
```

Reply:
I would consider getting the version number from `package.json` (crop to only major and minor version) to keep it up-to-date.

Reply:
I don't think we should add `lisk-js` version number.
Lisk-js APIClient is just a library and not actual client imo

Reply:
@shuse2 That APIClient is based on an engine/library/module `lisk-js`which have a specific version number. And for debugging and analytics its a valuable information.

Reply:
Let's arrange a meeting to discuss this.